docs/apache-airflow-providers-cncf-kubernetes/operators.rst

```diff
@@ -15,114 +15,1015 @@
     specific language governing permissions and limitations
     under the License.
 
+###########################
+KubernetesPodOperator Guide
+###########################
+
+.. contents:: :local:
 
 
 .. _howto/operator:KubernetesPodOperator:
 
-KubernetesPodOperator
-=====================
+Kubernetes Pod Operator "Hello World!"
+======================================
+
+- Creating Task using KubernetesPodOperator which prints "Hello World [HOSTNAME]".  Add ``in_cluster=False`` to
+  KubernetesPodOperator constructor when running examples with minikube.
```

Comment:
I'd recommend we push people towards KinD instead of minikube.

Reply:
ok.